python/cavity_flow.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+from navier_stokes_problem import StationaryNavierStokesProblem, VelocityBCType
+from grid_generator import square_cavity, HyperCubeBoundaryMarkers
```

Comment:
This error occurs because I renamed the method `square_cavity` to `hyper_cube` and forget to do this here. I fixed this in bfcc187. If you pull the repository again, the script should work.